Tools-Override/tests.targets

```diff
@@ -24,11 +24,16 @@
     <DefaultNoCategories Include="$(DefaultNoCategories)" />
     <UnsupportedPlatformsItems Include="$(UnsupportedPlatforms)"/>
   </ItemGroup>
+  
+  <ItemGroup>
+    <SupplementalTestData Include="$(RuntimePath)\xunit.console.netcore.exe" />
```

Comment:
I think this line is not required as the exe comes from the external depproj. I believe this was part of the change before individual test folders, will remove.